packages/mocks/src/getMockData.ts

```diff
@@ -1,13 +1,9 @@
-import { PathParams } from 'msw';
-
-import * as providers from './data/provider.json';
+import basicMocks from './definitions/basic/paths';
 
 /**
  * A mapping of API paths to their corresponding mock data.
  */
-const mockData: Record<string, object> = {
-  '/api/proxy/plugin/forklift-console-plugin/forklift-inventory/providers': providers,
-};
+const mockData: Record<string, object> = basicMocks;
```

Comment:
do you need this ?
can we use basic mocks inside the get mock data method ?

Reply:
let's treat it as temporary state - in the final version you should be able to switch between data sets - see `resolvePath` method

Reply:
ok